common/src/main/java/org/apache/gravitino/dto/requests/MetalakeCreateRequest.java

```diff
@@ -74,6 +74,6 @@ public MetalakeCreateRequest(String name, String comment, Map<String, String> pr
   @Override
   public void validate() throws IllegalArgumentException {
     Preconditions.checkArgument(
-        StringUtils.isNotBlank(name), "\"name\" field is required and cannot be empty");
+        StringUtils.isNotBlank(name), "\"name\" field is required. Please provide a valid name for the Metalake. It cannot be empty or only whitespace.");
```

Comment:
The issue with this change is that it changes the Java API, not the Gravitino CLI. In the Java API context, this error message is correct, in the CLI context, the message could be more helpful. The change needs to be made in teh CLI code not in the Java API code.